ui-v2/app/styles/base/components/buttons/layout.scss

```diff
@@ -48,3 +48,28 @@
   display: inline-block;
   box-sizing: border-box;
 }
+%split-button {
+  @extend %secondary-button;
+  padding: 0 8px !important;
+  position: relative;
+  height: 100%;
+}
+%split-button span::after {
+  @extend %as-pseudo;
+  position: absolute;
+  background-color: $gray-300;
+  width: 1px;
+  top: 0;
+  height: 103%;
+  margin-left: 8px;
+}
+%split-button::before {
+  @extend %as-pseudo;
+  height: 16px;
+}
+%sort-button::before {
+  @extend %with-sort-icon;
+  margin-top: 8px;
+  width: 16px;
+  height: 16px;
+}
```

Comment:
Wondering if there's some stuff that could go in `skin.scss` here, maybe from line 57 down as that is all to do with the pipe and the icon, don't worry too much though we can always hit that in another spring clean PR, or decide when we next chat, its no biggie.

Oh wow I just saw what you did with the `103%`, I'm guessing this is to do with making sure the top and bottom of the pipe meet up with the borders? I love the 103% btw, but I just has a thought that maybe a 'calc(100% + 3px)' might be better (if thats what this is for of course?)

Reply:
I'll move those lines over to `skin.scss`. 

calc(100% + 3px) is not the same as 103% so it extends too far down. The exact px match would be calc(100% + .5px). Which I think looks worse than the 103%. 🤔

Reply:
Update on the height. I had the sort matching the search before but I must have undone the change. Adding this change removes the need of a `height: 103%`.